python/lsst/ctrl/bps/wms/panda/conf_example/pipelines_check_idf.yaml

```diff
@@ -1,13 +1,17 @@
-pipelineYaml: "${OBS_SUBARU_DIR}/pipelines/DRP.yaml#processCcd"
+includeConfigs:
+- ${CTRL_BPS_DIR}/config/bps_idf.yaml
+
+operator: ${USER}
+project: dev
+campaign: quick
+pipelineYaml: "${OBS_LSST_DIR}/pipelines/imsim/DRP.yaml#step1"
 
 payload:
   payloadName: pipelines_check
-  butlerConfig: s3://butler-us-central1-panda-dev/hsc/butler.yaml
-  inCollection: HSC/calib,HSC/raw/all,refcats
-  dataQuery: "tract = 9615 and patch=30 and detector IN (10..11) and instrument='HSC' and skymap='hsc_rings_v1' and band in ('r')"
-  output: u/{operator}/pcheck_ebutler
+  output: "u/{operator}/{payloadName}"
```

Comment:
This is the default.  I don't object if you want to put it here to make it easy for someone to change it.

Reply:
I'd leave this here as I think we might want to change it in most DP0.2 runs.